tree_sitter_v/queries/highlights.scm

```diff
@@ -73,6 +73,7 @@
  "shared"
  "static"
  "struct"
+ "implements"
```

Comment:
Could this be added in alphabetical order?  It is _so_ much easier for humans to look for things in alpha order...

Reply:
I sorted all keywords. `spawn` was also out of order.